modules/hyprlock/hm.nix

```
@@ -6,7 +6,7 @@ with config.lib.stylix;
 
   config = lib.mkIf (config.stylix.enable && config.stylix.targets.hyprlock.enable) {
     programs.hyprlock.settings = {
-      background.path = config.stylix.image;
+      background.path = toString config.stylix.image;
```

Prevent the file path from getting garbage collected:

```suggestion
      background.path = "${config.stylix.image}";
```